core/src/test/scala/org/broadinstitute/dsde/rawls/billing/BillingProfileManagerDAOSpec.scala

```diff
@@ -147,12 +144,56 @@ class BillingProfileManagerDAOSpec extends AnyFlatSpec with MockitoSugar {
     val config = new MultiCloudWorkspaceConfig(true, None, None)
     val bpmDAO = new BillingProfileManagerDAOImpl(provider, config)
 
-    val profile = bpmDAO.createBillingProfile("fake", Right(coords), testContext)
+    val profile = bpmDAO.createBillingProfile("fake", Right(coords), Map.empty, testContext)
 
     assertResult(expectedProfile)(profile)
     verify(profileApi, times(1)).createProfile(ArgumentMatchers.any[CreateProfileRequest])
   }
 
+  it should "include an empty set of policy inputs if no policies are present" in {
+    val provider = mock[BillingProfileManagerClientProvider](RETURNS_SMART_NULLS)
+    val profileApi = mock[ProfileApi](RETURNS_SMART_NULLS)
+    val dummyProfile = new ProfileModel().id(UUID.randomUUID())
+    when(profileApi.createProfile(ArgumentMatchers.any[CreateProfileRequest])).thenReturn(dummyProfile)
+    when(provider.getProfileApi(ArgumentMatchers.eq(testContext))).thenReturn(profileApi)
+    val config = new MultiCloudWorkspaceConfig(true, None, None)
+    val coords = AzureManagedAppCoordinates(UUID.randomUUID(), UUID.randomUUID(), "fake_mrg")
+    val policies = Map[String, Map[String, String]]()
+    val bpmDAO = new BillingProfileManagerDAOImpl(provider, config)
+
+    val createProfileRequestCaptor = captor[CreateProfileRequest]
+    bpmDAO.createBillingProfile("fake", Right(coords), policies, testContext)
+
+    val expectedPolicies = new BpmApiPolicyInputs().inputs(
+      List.empty.asJava
+    )
+
+    verify(profileApi).createProfile(createProfileRequestCaptor.capture)
+    assertResult(expectedPolicies)(createProfileRequestCaptor.getValue.getPolicies)
+  }
+
+  it should "include any policies that are present" in {
+    val provider = mock[BillingProfileManagerClientProvider](RETURNS_SMART_NULLS)
+    val profileApi = mock[ProfileApi](RETURNS_SMART_NULLS)
+    val dummyProfile = new ProfileModel().id(UUID.randomUUID())
+    when(profileApi.createProfile(ArgumentMatchers.any[CreateProfileRequest])).thenReturn(dummyProfile)
+    when(provider.getProfileApi(ArgumentMatchers.eq(testContext))).thenReturn(profileApi)
+    val config = new MultiCloudWorkspaceConfig(true, None, None)
+    val coords = AzureManagedAppCoordinates(UUID.randomUUID(), UUID.randomUUID(), "fake_mrg")
+    val policies = Map("protected-data" -> Map[String, String]())
```

Comment:
Nit: it seems like it would be good to include a policy that has additional data, just to make sure that it gets passed through.

Reply:
Good call, I'll add that

Reply:
Adding this to the test uncovered a bug that using a `Map[String, String]` means a policy can't have additional data with a duplicate key which I believe is something we want to support for cases like group constraints with multiple groups. Replaced the `Map[String, String]` with a `List[(String, String)]`

Reply:
The bug is probably a non-issue for now since we're only setting the protected-data policy, but would need to be fixed if we ever wanted to support more policies

Reply:
Yes, we should be able to support policies like this:

```
"policies": [
        {
            "additionalData": [
                {
                    "group": "20231115_policy_test"
                },
                {
                    "group": "20231024_policy_test"
                }
            ],
            "name": "group-constraint",
            "namespace": "terra"
        }
    ]

Reply:
We've also seen this with regionality contraints--

```
const regionConstraintPolicy: WorkspacePolicy = {
  additionalData: [
    {
      'region-name': 'azure.eastus',
    },
    {
      'region-name': 'azure.westus2',
    },
    {
      'region-name': 'unknownRegion',
    },
  ],
  name: 'region-constraint',
  namespace: 'terra',
};